### PR TITLE
Update customer-key-manage.md

### PR DESCRIPTION
**Pull request by sathana87**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/customer-key-manage.md [hunk 1]
```diff
@@ -336,7 +336,7 @@ Remove-AzKeyVaultAccessPolicy -VaultName Contoso-O365EX-NA-VaultA1 -UserPrincipa
 
 ## Roll back from Customer Key to Microsoft managed Keys
 
-If you need to revert to Microsoft-managed keys, you can. When you offboard, your data is re-encrypted using default encryption supported by each individual workload. For example, Exchange Online supports default encryption using Microsoft-managed keys.
+If you need to revert to Microsoft-managed keys, you can in exchange. When you offboard, your data is re-encrypted using default encryption supported by each individual workload. For example, Exchange Online supports default encryption using Microsoft-managed keys. Roll back is not supported in SPO.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md:339]:
```suggestion
In Exchange Online, if you need to revert to Microsoft-managed keys, you can. When you offboard, your data is re-encrypted using default encryption supported by each individual workload. For example, Exchange Online supports default encryption using Microsoft-managed keys. In SharePoint, you can't revert to Microsoft-managed keys, rollback is not supported. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sathana87, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@sathana87 Gentle reminder

**Comment by yogkumgit** [comment]
@sathana87 Please update

**Comment by yogkumgit** [comment]
@sathana87 Please implement the suggested changes to initiate the merger of this PR. Thanks

File: microsoft-365/compliance/customer-key-manage.md (hunk 1 above)
Comment (KCCross) [reply microsoft-365/compliance/customer-key-manage.md:339]: Slight change to Johan's suggestion.

In Exchange Online, if you need to revert ...
In SharePoint, you can't revert to Microsoft-managed keys, roll back is not supported.

Comment (KCCross) [reply microsoft-365/compliance/customer-key-manage.md:339]: @sathana87

Comment (JohanFreelancer9) [reply microsoft-365/compliance/customer-key-manage.md:339]: Thank you @KCCross. I've changed my suggestion accordingly.

**Comment by yogkumgit** [comment]
@sathana87 Please implement the suggested changes, in the event of no response we may may have to close this PR. Thanks

**Comment by yogkumgit** [comment]
@sathana87 We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
Please feel free to re-open this issue if you see an improvement is required in the specific area of this documentation. We will certainly update this documentation with that valuable information. Thanks again, for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Pull request closed without merging**